test/foundry/unit/StakingRewardsV2/StakingV2Checkpointing.t.sol

```diff
@@ -354,7 +336,7 @@ contract StakingV2CheckpointingTests is DefaultStakingV2Setup {
             assertEq(value, previousTotal + amountToStake);
 
             // move beyond cold period
-            vm.warp(block.timestamp + stakingRewardsV2.cooldownPeriod());
+            vm.warp(block.timestamp + 2 weeks);
```

Comment:
how come we mostly remove the cooldown period in this test file but then sometimes we add back this warp forward 2 weeks?